yt_dlp/extractor/vimeo.py

```diff
@@ -217,6 +217,12 @@ def _parse_config(self, config, video_id):
                     'url': thumbnail,
                 })
 
+            thumbnail = video_data.get('thumbnail_url')
+            if thumbnail:
+                thumbnails.append({
+                    'url': thumbnail,
+                })
+
```

Comment:
I'd replace this block and the `video_data.get('thumbnail')` block above with the single line below:
```py
           thumbnails.extend(traverse_obj(video_data, (('thumbnail', 'thumbnail_url'), {'url': {url_or_none}})))
```

Reply:
fixed